test/simple.test.ts

```diff
@@ -0,0 +1,171 @@
+import {
```

Comment:
I would appreciate that we rename the test files so that we don't get all these image diffs in the test outputs.